sources/packages/backend/workflow/test/2022-2023/parttime-assessment/awards-amounts/parttime-assessment-awards-amount-CSGD.e2e-spec.ts

```diff
@@ -50,7 +50,7 @@ describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-awards-amount-CS
       calculatedAssessment.variables.dmnPartTimeAwardFamilySizeVariables
         .limitAwardCSGDIncomeCap,
     );
-    expect(calculatedAssessment.variables.federalAwardCSGDAmount).toBe(1800);
+    expect(calculatedAssessment.variables.federalAwardNetCSGDAmount).toBe(1800);
```

Comment:
There was a minor comment left in the hotfix PR but considering the urgency, I was ok with that PR not making the change.

Can we make the change here?

https://github.com/bcgov/SIMS/pull/4399/files#r1972348828